src/poliastro/bodies.py

```diff
@@ -52,9 +66,68 @@ def _repr_latex_(self):
 
         """
         return self.__str__()
+    def calculate_soi(self):
+        '''Calculate the radius of the sphere of influence.
+        The body must have a parent body and an orbit defined'''
+        if self.parent == None:
+            warnings.warn('Unable to calculate SOI without valid parent body')
+        elif self.orbit == None:
+            warnings.warn('Unable to calculate SOI without valid orbit assigned')
+        else:
+            self.soi= self.orbit.a * (self.k / self.parent.k) ** (2/5)
+
+_data_path = os.path.join(os.path.dirname(__file__), "bodies_data.txt")
+with open(_data_path, 'r', encoding='utf-8') as _raw_data :
```

Comment:
Yes, I agree with having it built into a function. I will search how to use those libraries, but it will require to expand the dependencies of Poliastro, are you ok with it?